File: VB/CustomDescendants/MyBandedGridViewInfoRegistrator.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Base.Handler

Namespace TreeListCustomization
	Friend Class MyBandedGridViewInfoRegistrator
		Inherits BandedGridInfoRegistrator
		Public Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyBandedGridView"
			End Get
		End Property
		Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
			Return New MyBandedGridView(grid)
		End Function
	End Class
End Namespace

File: CS/CustomDescendants/MyBandedGridViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Base.Handler;

namespace TreeListCustomization
{
    class MyBandedGridViewInfoRegistrator : BandedGridInfoRegistrator
    {
        public override string ViewName { get { return "MyBandedGridView"; } }
        public override BaseView CreateView(GridControl grid) { return new MyBandedGridView(grid); }
    }
}
